res/qml/StemKnob.qml

```diff
@@ -6,62 +6,139 @@ import "Theme"
 Rectangle {
     id: root
 
-    property alias knob: knob
+    property alias knob: volume
 
     required property string group
     required property string label
     required property color stemColor
     required property int index
 
-    width: 56
-    height: 56
-    radius: 5
-    color: stemColor
+    readonly property string fxGroup: `[QuickEffectRack1_${group.substr(0, group.length-1)}Stem${index + 1}]]`
```

Comment:
Please note that `qml_formatter` is currently broken. 
I have made [a fix](https://github.com/qarmin/qml_formatter/pull/6) for it but the repo hasn't been looked at for 2 years, so not sure this is still being maintained. Happy to setup my instead, or maybe we could even adopt that project in the MixxxDJ organisation?

Reply:
Ignore me, this was merged already! Will update and fix formatting later.

Reply:
Will you create a PR to update our CI?

Reply:
Created #13530 with the fix + some other updates for pre-commit